object_manager.go

```diff
@@ -752,6 +753,39 @@ func (objMgr *ObjectManager) GetGridInfo() ([]Grid, error) {
 	return res, err
 }
 
+// CreateZoneAuth creates zones and subs by passing fqdn
+func (objMgr *ObjectManager) CreateZoneAuth(fqdn string, ea EA) (*ZoneAuth, error) {
```

Comment:
is this method not needed to be part of interface `IBObjectManager` ?

Reply:
IBObjectManager updated